frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/prefill-vm-template-state-update.ts

```diff
@@ -177,18 +180,31 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
         }
       }
 
+      let type = diskWrapper.isFirstBootableDevice()
+        ? VMWizardStorageType.PROVISION_SOURCE_TEMPLATE_DISK
+        : VMWizardStorageType.TEMPLATE;
+
+      if (
```

Comment:
what happens if somebody sets a boot order to win tools disk? Maybe we should not allow it and delete the boot order here and update a boot order select component? 

WDYT? @yaacov  @glekner

Reply:
We do not enforce checking for bootability of a device, and allow users to set any device as a boot-able device, even if it's unbootable,  the reasons behind this policy are:
a - we can't reliably test for device boot-ability.
b - if a device is unbootable, the system just go and try the next one in the list.
c - most devices are unbootable (except one), so we trust the user to set only the one bootable device as the first and only boot device.

MHO is that we do not need to treat `virtualization guest cd` differently than other unbootable devices.

Reply:
ok then
In that case we will have a wrong behaviour when somebody creates a template with wintools image with bootorder 1.

meaning: there is conflict between `PROVISION_SOURCE_TEMPLATE_DISK` and `WINDOWS_GUEST_TOOLS_TEMPLATE` types

maybe we should add a small TODO since it is not a big issue? not sure if we can fix this easily right now

i guess we should revisits this issue once we start adding boot order component to the wizard and revamp all the boot order logic inside the wizard.

now, it makes the most sense to me to replace the `VMWizardStorageType`  with a set of flags like `template`, `defaultProvisionStorage`, `guestTools`